### PR TITLE
50 create choosevaluecomponent

### LINKED ISSUE #50: create ChooseValueComponent
image

### PR DESCRIPTION
**Pull request by Zaai90**
closes #50

### CODE REVIEW

File: screens/ChoresScreen.tsx [hunk 1]
```diff
@@ -11,17 +13,29 @@ type Props = NativeStackScreenProps<ChoreStackParams>;
 
 const ChoresScreen = ({ navigation }: Props) => {
   const chores = useAppSelector(selectChores);
+  const [value, setValue] = React.useState(1);
```
Comment (Zaai90) [inline screens/ChoresScreen.tsx:16]:
```suggestion
```

File: screens/ChoresScreen.tsx [hunk 2]
```diff
@@ -11,17 +13,29 @@ type Props = NativeStackScreenProps<ChoreStackParams>;
...
 
   return (
-    <View style={{ flex: 1 }}>
+    <View style={{ flex: 1, padding: 10 }}>
```
Comment (Zaai90) [inline screens/ChoresScreen.tsx:19]:
```suggestion
```

File: screens/ChoresScreen.tsx [hunk 3]
```diff
@@ -11,17 +13,29 @@ type Props = NativeStackScreenProps<ChoreStackParams>;
...
           <ChoreItem chore={chore} />
         </Pressable>
       ))}
+      {/* //Städa */}
+      <ItemContainer>
+        <ValuePicker label={"Poäng"} value={value} min={1} steps={1} max={10} showBadge={true} unit={"Poäng"} onChange={setValue} />
+      </ItemContainer>
+      <ItemContainer>
+        <ValuePicker label={"Intervall"} subLabel={"Testing Testing"} value={value} min={1} steps={1} max={31} unit={"Dagar"} onChange={setValue} />
+      </ItemContainer>
```
Comment (Zaai90) [inline screens/ChoresScreen.tsx:32]:
```suggestion
```

File: screens/ChoresScreen.tsx [hunk 4]
```diff
@@ -11,17 +13,29 @@ type Props = NativeStackScreenProps<ChoreStackParams>;
...
     </View>
   );
 };
 
+const ItemContainer = styled.View`
+  margin: 5px;
+`;
+
```
Comment (Zaai90) [inline screens/ChoresScreen.tsx:40]:
```suggestion
```

**Review by Zaai90 — COMMENTED** [review]
cleanup

File: screens/ChoresScreen.tsx [hunk 5]
```diff
@@ -13,7 +15,6 @@ const ChoresScreen = ({ navigation }: Props) => {
   const chores = useAppSelector(selectChores);
 
   return (
-    <View style={{ flex: 1 }}>
       <Text>ChoresScreen</Text>
```
Comment (Zaai90) [inline screens/ChoresScreen.tsx:18]:
```suggestion
<View style={{ flex: 1 }}>
      <Text>ChoresScreen</Text>
```

File: screens/ChoresScreen.tsx [hunk 6]
```diff
@@ -2,7 +2,9 @@ import { NativeStackScreenProps } from "@react-navigation/native-stack";
 import React from "react";
 import { Pressable, View } from "react-native";
 import { Text } from "react-native-paper";
+import styled from "styled-components/native";
 import ChoreItem from "../components/ChoreItem";
+import ValuePicker from "../components/ValuePicker";
 import { ChoreStackParams } from "../navigation/ChoreStackNavigator";
```
Comment (Zaai90) [inline screens/ChoresScreen.tsx:8]:
```suggestion
import ChoreItem from "../components/ChoreItem";
import { ChoreStackParams } from "../navigation/ChoreStackNavigator";
```

File: screens/ChoresScreen.tsx [hunk 7]
```diff
@@ -13,8 +13,9 @@ const ChoresScreen = ({ navigation }: Props) => {
   const chores = useAppSelector(selectChores);
 
   return (
+<View style={{ flex: 1 }}>
     <View style={{ flex: 1 }}>
-      <Text>ChoresScreen</Text>
+        <Text>ChoresScreen</Text>
```
Comment (Zaai90) [inline screens/ChoresScreen.tsx:18]:
```suggestion
      <Text>ChoresScreen</Text>
```

**Review by 4l3x91 — APPROVED** [review]
e=mc2

**Pull request merged**